easybuild/easyconfigs/m/MDSplus/MDSplus-7.153.3-GCCcore-13.2.0.eb

```diff
@@ -0,0 +1,74 @@
+easyblock = 'ConfigureMake'
+
+name = 'MDSplus'
+version = '7.153.3'
+
+homepage = 'http://mdsplus.org/'
+description = """MDSplus is a set of software tools for data acquisition
+  and storage and a methodology for management of complex scientific data."""
+
+toolchain = {'name': 'GCCcore', 'version': '13.2.0'}
+toolchainopts = {'pic': True}
+
+sources = [{
+    'filename': SOURCELOWER_TAR_XZ,
+    'git_config': {
+        'url': 'https://github.com/' + name,
+        'repo_name': '%(namelower)s',
+        'commit': 'stable_release-' + version.replace('.', '-'),
+        'recursive': True,
```

Comment:
```suggestion
        'recursive': True,
        # Explicitly do not keep the .git directory to avoid pulling submodules
        # the git commands will fail during the bootstrap, but this is fine for the build
        'keep_git_dir': False
```